test/MenuItemSpec.js

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import ReactTestUtils from 'react/lib/ReactTestUtils';
 import ReactDOM from 'react-dom';
 import tsp from 'teaspoon';
+
 import MenuItem from '../src/MenuItem';
 
 import { shouldWarn } from './helpers';
 
-describe('MenuItem', () => {
+describe('<MenuItem>', () => {
```

Comment:
`.only()` does a grep, which ends up matching other tests... I don't know, maybe I'll back this out and just hack it to e.g. `describe('XXXMenuItem')` when testing anyway

https://github.com/mochajs/mocha/pull/1807